include/cantera/oneD/Flow1D.h

```diff
@@ -670,41 +674,166 @@ class Flow1D : public Domain1D
 
     //! @name Convective spatial derivatives
     //!
-    //! These use upwind differencing, assuming u(z) is negative
+    //! These methods use upwind differencing to calculate spatial derivatives
+    //! for velocity, species mass fractions, and temperature. Upwind differencing
+    //! is a numerical discretization method that considers the direction of the
+    //! flow to improve stability.
     //! @{
+
+    /**
+     * Calculates the spatial derivative of velocity V with respect to z at point j
+     * using upwind differencing.
+     *
+     * The general formula for a variable, Var, is:
+     * @f[
+     *   \frac{\partial Var}{\partial z} \bigg|_{j} \approx \frac{Var(x, j_{\text{loc}}) -
+     *   Var(x, j_{\text{loc}} - 1)}{m_{\text{dz}}[j_{\text{loc}} - 1]}
+     * @f]
+     *
```

Comment:
The notation here is getting rather confused by mixing mathematical conventions and quantities that appear in the code. On the basis of the "don't repeat yourself" principle, I think I'd prefer to leave the description here as just saying that the method uses upwind differencing, and link back to the new page in the Sphinx docs. As explained in the [developer docs](https://cantera.org/dev/develop/doc-formatting.html#useful-doxygen-syntax), you can link to that page from Doxygen as something like : ```[upwind differencing](../reference/onedim/discretization.html#upwinding)``` (double checking the anchor name after any updates to that page, of course)